duckduckgo/iOS
# Setting layout changes

Task/Issue URL: https://app.asana.com/0/414235014887631/1202360923697985/f
CC: @THISISDINOSAUR 

**Description**:
Changes the Settings layout and adds an Autofill section (hidden)

**Steps to test this PR**:

1. Check if the layout is matching the [figma file](https://app.asana.com/0/414235014887631/1202360923697985/f)
2. Check if the cells are working as expected

----

1.  Change the code [here](https://github.com/duckduckgo/iOS/blob/49bb17e045ea117f6c98701ab5e47fa0853035c8/DuckDuckGo/SettingsViewController.swift#L81) to return true 
2.  Check if the layout is matching the [figma file](https://app.asana.com/0/414235014887631/1202360923697985/f)
3. Check if the cells are working as expected


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
